geoengine/raster_workflow_rio_writer.py

```diff
@@ -168,20 +168,32 @@ def __create_new_dataset(self, query: QueryRectangle):
             **self.rio_kwargs
         )
 
+        for i, b in enumerate(self.bands, start=1):
+            b_n = b.name
+            b_m = str(b.measurement)
+            rio_dataset.update_tags(i, band_name=b_n, band_measurement=b_m)
+
         self.current_dataset = rio_dataset
 
-    async def query_and_write(self, query: QueryRectangle):
+    async def query_and_write(self, query: QueryRectangle, skip_empty_times=True):
```

Comment:
is it skip_empty_tiMEs or skip_empty_tiLes?

Reply:
we skip times. Tiles are not really of interest here since we write all the tiles in one GDAL dataset anyway.